apps/rich-text-app/package.json

```diff
@@ -20,7 +20,7 @@
     "upload-ci": "contentful-app-scripts upload --ci --bundle-dir ./build  --organization-id $CONTENTFUL_ORG_ID --definition-id $CONTENTFUL_APP_DEF_ID --token $CONTENTFUL_ACCESS_TOKEN"
   },
   "eslintConfig": {
-    "extends": "react-app"
```

Comment:
We shouldn't use relative paths here. Otherwise, people cannot copy that folder without changing the package.json. The example should not depend on anything outside of its folder.